### DISCUSSION TITLE
How to ensure one-way sync?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [bpozdena/OneDriveGUI#136] [Q&A] [answered]
I don't trust OneDriveGUI entirely yet, so I'd like to know how to ensure that it's solely performing a one-way sync — after all, if the synchronization process breaks somehow, it might sync the breakage back to the server.

**Comment by roke-julian-lockhart** [comment]
Filed at [`issues/140`](https://github.com/bpozdena/OneDriveGUI/issues/140#issue-1999792556), because it's probably not implemented.

**Reply by roke-julian-lockhart** [reply]
See [`issues/140#issuecomment-1816997863`](https://github.com/bpozdena/OneDriveGUI/issues/140#issuecomment-1816997863), then [`orgs/community/discussions/142061`](https://github.com/orgs/community/discussions/142061#discussion-7343560)…

**Comment by bpozdena** [comment]
If you are unsure how the sync will behave with specific settings, you can enable the [Dry run](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#testing-your-configuration) option, which will just simulate the sync process without actually moving any files around. 

![image](https://github.com/bpozdena/OneDriveGUI/assets/24818591/3d010db8-d6fd-497e-8404-b6f9e7bcb716)

**Reply by roke-julian-lockhart** [reply]
@bpozdena, is that equivalent to `--download-only`, via the CLI? [^1]

[^1]: [`#discussioncomment-7602761`](https://github.com/bpozdena/OneDriveGUI/discussions/136#discussioncomment-7602761)

**Comment by bpozdena** [comment, marked as answer]
The GUI will use --download-only when you enable 'Download only' in the GUI.

If you hover your mouse cursor over any GUI option, you'll get a basic tool tip. For more details, please check the onedrive client documentation.
